en/ch06.md

```diff
@@ -4,24 +4,41 @@ The `Settings` tab provides tools for customizing configuration options.
 ### Wallet
 ![settings-wallet](media/black_settings-wallet.png)
 
+(1) **Close this wallet:** Click here to safely close the current wallet  
+(2) **Create a view-only wallet:** Click here to generate a view-only (or audit) wallet, which is capable to see existing outgoing transactions and all incoming transactions. See this [user-guide](https://getmonero.org/resources/user-guides/view_only.html) for more informations.  
+(3) **Show seed & keys:** This button will redirect you to the [Seed and keys](#seed-and-keys) tab.  
+(4) **Rescan wallet balance:** Click here to rescan the blockchain for all the transaction. *Note: You will lose any information not stored on the blockchain (this includes destination addresses, tx secret keys, tx notes, etc.)*  
+(5) **Change wallet password:** Click here to change your password.
+
+### Layout
+![settings_layout](media/black_settings-layout.png)
+
+The `layout` tab is where you control the GUI wallet behavior. Most important options are:
+
+(1) **Hide balance:** If you are running your wallet in public, you may want to hide it's balance. This could be useful for point of sales.  
+(2) **Enable Payment ID:** Payment ID are disabled by default. Click here to enable it. This can be useful to make transfers to Exchanges, which requires a Payment ID.  
+(3) **Lock wallet:** Set this to automatically lock your wallet after N minutes of inactivity. This would avoid anyone to use your wallet while you are away from keyboard.
```

Comment:
> This would avoid anyone to use your wallet while you are away from keyboard.

This ensures no one is able to use your wallet while you are away from keyboard.

Reply:
ok